GitCommands/Git/GitGpgController.cs

```diff
@@ -232,13 +232,13 @@ public string GetCommitVerificationMessage(GitRevision revision)
                 return null;
             }
 
-            var module = GetModule();
+            // '--raw' returns info in stderr
             GitArgumentBuilder args = new("verify-tag")
             {
                 { raw, "--raw" },
                 tagName
             };
-            return module.GitExecutable.GetOutput(args);
+            return GetModule().GitExecutable.Execute(args).StandardError;
```

Comment:
Breaks tests.  Need to tweak StageOutput system.  Probably need to have an overload that lets us set output and error outputs.  

UnitTests\GitCommands.Tests\Git\Gpg\GitGpgControllerTests
Summary: Passed: 16, Failed: 5, Ignored: 0

Reply:
The test was incorrect.

Reply:
I agree but I was pointing out the fact that stageoutput needs to be able to specify error and output values to really make  this work or we do something like we know  command x outputs in error so flip streams with a bool flag in the whatnot methods.  

```cs
Theexe.GetOutput(args,FlipStreams:true);
```

Reply:
Moved to #10262